### PR TITLE
Wee little school finder bugs

### PR DESCRIPTION
**Pull request by DFurnes**
# Changes
- Fix page jumping to top when the user clicks "Show More".
- Improve appearance of "Show More" button based on design feedback.
# Screenshot

screen shot 2014-11-07 at 11 45 28 am

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/js/campaign/SchoolFinder.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ define(function(require) {
     }
 
     if(canShowMore) {
-      $el.append("<p class='legal'><a href='#' class='secondary js-schoolfinder-showmore'>Show More Results</a></p>");
```
Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/js/campaign/SchoolFinder.js:50]: Lol. It was _classed_ as legal @_@

Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/js/campaign/SchoolFinder.js:50]: :necktie:

File: lib/themes/dosomething/paraneue_dosomething/scss/content/modal-schoolfinder.scss [hunk 2]
```diff
@@ -59,6 +59,27 @@
...
+      color: $med-gray;
+      font-size: 16px;
+      position: absolute;
+      top: 1px;
+      right: 6px;
+      @include transform(rotate(0));
+    }
+  }
+}
+
```
Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/modal-schoolfinder.scss:82]: Side note, if `.legal` and `.schoolfinder-showmore` technically have very similar styles, we could generalize a class and then `@extend` it to both. Maybe be a future refactor if that makes sense?

**Comment by weerd** [comment]
:dancer: :+1:

File: lib/themes/dosomething/paraneue_dosomething/scss/content/modal-schoolfinder.scss (hunk 2 above)
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/modal-schoolfinder.scss:82]: Totally. :+1:

**Pull request merged**